nano/templates/firealarm.tmpl

```diff
@@ -6,7 +6,7 @@ The current alert level is:
 	<span style='color: rgb(13, 186, 212);font-weight: bold;'>Blue</span>
 {{/if}}
 {{if data.alertLevel == 'yellow'}}
-	<span class='yellow'>Yellow</span>
+	<span style='color: rgb(255, 255, 0);font-weight: bold;'>Yellow</span>
```

Comment:
Is #FF0000 really the best color? Sounds like it'd burn your eyes out, maybe go with something dimmer, maybe even the light color?

Reply:
Specifically, something with a lower lightness and saturation.

Reply:
`rgb(225, 225, 125)` is the yellow light color, and I think it would be easier on the eyes.

Reply:
fyi, this is `#FFFF00`, not `#FF0000`.